crates/holochain/src/core/sys_validate.rs

```diff
@@ -311,7 +311,7 @@ pub fn check_prev_author(action: &Action, prev_action: &Action) -> SysValidation
 pub fn check_prev_timestamp(action: &Action, prev_action: &Action) -> SysValidationResult<()> {
     let t1 = prev_action.timestamp();
     let t2 = action.timestamp();
-    if t2 > t1 {
+    if t2 >= t1 {
```

Comment:
should be fine; we have action_seq to determine order of actions

Reply:
There are tests for this though, can a new one be added to assert this new boundary condition?

Reply:
I would still like to see a test that checks the boundary condition here. There is already a test that is checking the negative of this condition but with a strict inequality, there wasn't a boundary here. If that test could be updated to include a check with equal timestamps or a new test could be added based on the existing one that would be great. I took quite a bit of time to get detailed tests in place for this workflow and it'd be good to keep the coverage of conditions high. Given that the tests passed before and after this change, if somebody puts it back without thinking too hard about why it was changed, we won't have any obstacle there to make it obvious this was done for a good reason